lib/client-assets.php

```diff
@@ -187,6 +201,29 @@ function gutenberg_register_packages_scripts( $scripts ) {
 	// else (for development or test) default to use the current time.
 	$default_version = defined( 'GUTENBERG_VERSION' ) && ! SCRIPT_DEBUG ? GUTENBERG_VERSION : time();
 
+	// Clear the file cache for HMR.
+	if ( SCRIPT_DEBUG ) {
+		clearstatcache( true, gutenberg_dir_path() . 'build/🔥hot.js' );
+	}
+
+	// Register the runtime script.
+	gutenberg_override_script(
+		$scripts,
+		'wp-runtime',
```

Comment:
Do you think it should be included in the WordPress core, too? Otherwise, it could be a regular script registration and the script handle should be more `gutenberg-packages-runtime`.

Reply:
If it isn’t included in WP core, there is no value in the overhead of `gutenberg_override_script` which unregisters core scripts first.

Reply:
I see. I think this is worth getting into core too. I need to get familiar with core's webpack config first though 😅.